config/packages/security.yaml

```diff
@@ -1,7 +1,7 @@
 parameters:
     sylius.security.admin_regex: "^/admin"
     sylius.security.api_regex: "^/api"
-    sylius.security.shop_regex: "^/(?!admin|api/.*|api$|media/.*)[^/]++"
+    sylius.security.shop_regex: "^/(?!admin|api|new-api/.*|api$|media/.*)[^/]++"
```

Comment:
```suggestion
    sylius.security.shop_regex: "^/(?!admin|new-api|api/.*|api$|media/.*)[^/]++"
```

Reply:
Btw. it feels like we should write down why shop regex should match `/api`, `/api/something` but not `/api/` - I've already forgotten.